stdlib/asm/crypto/stark/verifier.masm

```diff
@@ -46,6 +46,13 @@ const.LDE_SIZE_PTR=4294967292             # Z_PTR + 1
 #!   - The maximal allowed degree of the remainder polynomial is 7.
 #!   - Only the input and output stacks, assumed of fixed size equal to 16, are handled in regards
 #!   to public inputs.
+#!   - There are two trace segments, main and auxiliary. It is assumed that the main trace segment
+#!   is 72 columns wide while the auxiliary trace segment is 9 columns wide.
+#!   - The OOD evaluation frame is composed of two rows, current and next, each composed of 72 elements
+#!   representing the main trace portion and 9 elements for the auxiliary trace one.
+#!   - To boost soundness, the protocol is run on a quadratic extension field and this means that
+#!    the OOD evaluation frame is composed of elements in a quadratic extension field i.e. tuples.
+#!    Similarly, elements of the auxiliary trace are quadratic extension field elements.
```

Comment:
This is great, thanks! I'd also add a comment above the STARK verifier test that says something like "Changes to MidenVM may cause this test to fail due to hard-coded assumptions noted in the STARK verifier implementation in stdlib/asm/crypto/stark/verifier.masm"

Reply:
Makes sense! I have included a small note in the corresponding test.